src/Symfony/Component/HttpKernel/HttpCache/HttpCache.php

```diff
@@ -78,6 +78,9 @@ class HttpCache implements HttpKernelInterface, TerminableInterface
      *                            the cache can serve a stale response when an error is encountered (default: 60).
      *                            This setting is overridden by the stale-if-error HTTP Cache-Control extension
      *                            (see RFC 5861).
+     *
+     *   * terminate_on_cache_hit Specifies if the kernel.terminate event should be dispatched even when the cache
+     *                            was hit (default: true).
```

Comment:
i would explain a little bit why this option exists.

```suggestion
     *                            was hit (default: true).
     *                            Unless your application needs to process events on cache hits, it is recommended to set this to false to avoid having to bootstrap the Symfony framework on a cache hit.
```